content/courses/native-onchain-development/hello-world-program.md

```diff
@@ -13,156 +13,152 @@ description:
 
 ## Summary
 
-- Native Solana programs have a single **entry point** to process instructions
-- A program processes an instruction using the **program_id**, list of
-  **accounts**, and **instruction_data** included with the instruction
+- Native Solana programs have a single **entry point** to process instructions.
+- A program processes an instruction using the **program_id**, a list of
+  **accounts**, and **instruction_data** included with the instruction.
 
 ## Lesson
 
 <Callout type="info">
-The following guide assumes you are familiar with Solana program basics - if not, check out [introduction to onchain programming](/developers/courses/onchain-development/intro-to-onchain).
+
+The following guide assumes you are familiar with Solana program basics. If not,
+check out
+[Introduction to Onchain Programming](/content/courses/onchain-development/intro-to-onchain.md).
 </Callout>
 
-This lesson will give you a basic introduction to writing and deploying a Solana
-program using the Rust programming language, without any framework. This gives
-you greater control, but also requires you to perform much of the basic work of
-creating an onchain program yourself.
+This lesson will introduce you to writing and deploying a Solana program using
+the Rust programming language without any framework. This approach offers
+greater control but requires you to handle much of the foundational work of
+creating an on-chain program yourself.
```

Comment:
'onchain' is one word. See https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#prose and fix everywhere.